### ISSUE TITLE
Interactive learning Crashes with Reminders

### ISSUE THREAD

**Issue by tahamr83** [RasaHQ/rasa_core#1820]
**Rasa Core version**:
0.13.3
**Python version**: 
3.6.6
**Operating system** (windows, osx, ...):
Mac OS
**Issue**:
Interactive Learning crashes when a scheduled reminder is executed. After the reminder is executed and the user enters an utterance. After NLU validation the program crashes. Apparently it is unable to print the chat history table and _plot_trackers() after the reminder's execution. 

(I was able to almost bypass it by returning a '\n' string for _as_md_message() if parse data has a None for the 'text' field. But then the _plot_trackers() method crashed which I by passed by simply commenting out. )

**Content of domain file** (if used & relevant):
```yaml

```

**Comment by akelad** [comment]
Thanks for raising this issue, @tmbo  will get back to you about it soon.

**Comment by tmbo** [comment]
Yes, this is a known limitation - It might work after the switch to asyncio - not sure though there might be another underlying issue.
